src/pybind/mgr/dashboard/controllers/cluster_configuration.py

```diff
@@ -3,14 +3,34 @@
 
 import cherrypy
 
-from . import ApiController, RESTController
+from . import ApiController, RESTController, ControllerDoc, EndpointDoc
 from .. import mgr
 from ..security import Scope
 from ..services.ceph_service import CephService
 from ..exceptions import DashboardException
 
 
+FILTER_SCHEMA = [{
+    "name": (str, 'Name of the cluster'),
+    "type": (str, 'Cluster type'),
+    "level": (str, 'level of info'),
+    "desc": (str, 'description of the configuration'),
+    "long_desc": (str, 'elaborated description'),
+    "default": (str, 'Default value set for the cluster'),
+    "daemon_default": (str, 'Default Daemon associated to the cluster'),
+    "tags": ([str], 'tags associated with the cluster'),
+    "services": ([str], 'services associated with the cluster'),
+    "see_also": ([str], 'additional info'),
```

Comment:
The field names other related config options... maybe:
```suggestion
    "see_also": ([str], 'Related config options'),
```
?